File: WorldFeed/Server/src/WorldFeed.Common/DTO/Webhooks/GetAccountActivityWebhookEnvironmentsResultDTO.cs
namespace WorldFeed.Common.DTO.Webhooks
{
    using WorldFeed.Common.Public.Models.Interfaces.DTO.Webhooks;

    public class GetAccountActivityWebhookEnvironmentsResultDTO : IGetAccountActivityWebhookEnvironmentsResultDTO
    {
        public IWebhookEnvironmentDTO[] Environments { get; set; }
    }
}

File: WorldFeed/Client/src/app/core/Core/DTO/Webhooks/GetAccountActivityWebhookEnvironmentsResultDTO.ts
import {IGetAccountActivityWebhookEnvironmentsResultDTO} from "../../../Public/Models/Interfaces/DTO/Webhooks/IGetAccountActivityWebhookEnvironmentsResultDTO";
import {IWebhookEnvironmentDTO} from "../../../Public/Models/Interfaces/DTO/Webhooks/IWebhookEnvironmentDTO";

export class GetAccountActivityWebhookEnvironmentsResultDTO implements IGetAccountActivityWebhookEnvironmentsResultDTO {
  public environments: IWebhookEnvironmentDTO[];
}
